dev-app/src/main/java/com/vincentmasselis/devapp/MainActivity.kt

```diff
@@ -1,12 +1,19 @@
 package com.vincentmasselis.devapp
 
+import android.Manifest
 import android.bluetooth.BluetoothManager
 import android.content.Context
+import android.os.Build
 import android.os.Bundle
+import android.util.Log
 import androidx.appcompat.app.AppCompatActivity
+import com.vincentmasselis.rxbluetoothkotlin.NeedBluetoothConnectPermission
+import com.vincentmasselis.rxbluetoothkotlin.NeedBluetoothScanPermission
+import com.vincentmasselis.rxbluetoothkotlin.NeedLocationPermission
 import com.vincentmasselis.rxbluetoothkotlin.rxScan
 import com.vincentmasselis.rxuikotlin.disposeOnState
 import com.vincentmasselis.rxuikotlin.utils.ActivityState
+import java.util.logging.ErrorManager
```

Comment:
I've doubts this import is revelant for this MR, but I've some ideas why you make this little mistake